src/components/HomePage/AccountHeader/AccountHeader.tsx

```diff
@@ -18,10 +30,10 @@ const AccountHeader = (props: Props) => {
         </div>
       </div>
       <div className="actions">
-        <Button primary inverted>
+        <Button primary inverted onClick={handleChangeAlias}>
```

Comment:
i think this could be simplified to `<Button primary inverted href={BUILDER_URL + '/names'} target="_blank" />` instead of using callbacks and `window.open`

Reply:
Wow!!! I didn't know about it. Fixing!